### ISSUE TITLE
Change color of popups

### ISSUE THREAD

**Issue by HolbyFPV** [toby-j/jetbrains-github-dark-theme#3]
Thanks for the theme :)

I am having some small issues with it that I was unable to resolve through the Intelij settings.  I think they need to be resolved at the theme level.

![image](https://github.com/toby-j/jetbrains-github-dark-theme/assets/61704528/873160af-c02b-43fc-8349-8b128bbc3f54)

This background of the error popup is the same color as the background.  Can this be changed to a lighter grey?  

My second request is the parameter info popup.  It's hard to see what parameter I'm actively typing.  The bold is too hard to see.  Can the entire color be changed so it's more obvious?  

![image](https://github.com/toby-j/jetbrains-github-dark-theme/assets/61704528/4b74c97a-33da-4704-9985-23772b64ce26)

**Comment by toby-j** [comment]
Trying to figure out why your highlighted text is green... what languages are you using?

**Comment by HolbyFPV** [comment]
I ended up making my own theme in an effort to figure this out.  I forgot to close this issue, my bad.  
https://plugins.jetbrains.com/plugin/22825-holby-dark

Sadly after all that work, I was unable to make the original change that I was hoping for.  The closest I got was 
```
"ui": {
    "CompletionPopup": {
      "matchForeground": "accentColor"
    },
}
```
This does not work with UE5 C++ LSP, which is primary use for Rider.  I believe this rule was working Java and maybe some other languages.  Just none that I use.
